docs/developers.rst

```diff
@@ -43,16 +43,20 @@ Publish a release
 
 To publish a new release on `PyPI`_, make sure the changelog is up to date
 and make sure you bumped the module version in ``setup.py``. Tag master
-at the version. For example, something like::
+as that version. For example, something like::
 
     git tag 0.0.5
     git push --tags
 
-Run this from the repository root to publish on `PyPI`_::
+Make sure you have `twine`_ installed, and then run this from the repository
+root to publish on `PyPI`_ as both a source distribution and wheel::
 
-    python setup.py sdist register upload
+    rm -rf dist/*
```

Comment:
doesn't `python setup.py clean` do this? The command could be `setup.py clean sdist bdist_wheel`